docs/getting-started/user-guides/using-paymetheus.md

```diff
@@ -0,0 +1,151 @@
+and click the submit button.  Your multisig script and P2SH address will be
+generated and you will be redirected to the tickets page.
+
+Next, import the pool's 1-of-2 multisignature script (on the Scripts tab).  This allows the funds to
+be recovered in case of pool failure.
+
+![Importing a script](/img/Paymetheus-import-script.png)  
+
+Finally, tickets can be purchased.  Select the "Using a pool?" checkbox and copy
+the values from the pool.  When all fields are set and if there is enough input
+value is available, the purchase button will become enabled and tickets may be
+purchased.
+
+NOTE: While you can purchase tickets using Paymetheus, it cannot vote for you so at this time you must use a pool.
+If you would prefer to solo mine, check the [Getting Started](/getting-started/overview.md) 
+and [PoS Mining](/mining/proof-of-stake.ms) guides for more information.
+
+![Purchasing tickets](/img/Paymetheus-ticket-purchasing.png)  
+
+---
+
+## **Request Payment** ##
+This is where you can generate wallet addresses to give to other people so they can
+send you DCR. Simply choose the account you want funds to go to and press Generate Address.
+Copy the address (it's the top line that starts with Ds) and share that with the other person.
+The public key below is what is passed to the blockchain. You don't need to worry about it.
+Decred addresses can be used as many times as you want, but for privacy reasons it's best
+to generate a new one for each transaction. There's around 1.4E59 (that's 1.4 followed by 59 zeroes)
+addresses available so you don't need to worry about running out.  
+
+![Request Payment](/img/Paymetheus-receive.png)  
+
+---
+
+
+## **Transaction History** ##
+This tab shows a list of all transactions that occured. The transaction hash can be used with the
+[block explorer](/getting-started/using-the-block-explorer.md) to see more information about the transaction.  
+
+![Transaction History](/img/Paymetheus-transactions.png)  
+
+---
+
+
+## **Stake Mining** ##
+This tab shows some statistics on the PoS network:  
+
+Item                         | Description
+-----------------------------|------------------------------------------------------------
+Number of live tickets       | This is the number of tickets that are eligible for voting.
+Number of tickets in mempool | This is number of tickets waiting to enter the voting pool.
+Ticket difficulty            | The cost of a ticket (refunded on ticket vote/expiry).
+Owned tickets in mempool     | Your tickets in the mempool.
+Owned live tickets           | Your tickets eligible for voting.
+Owned immature tickets       | Tickets entering the voting pool must wait to mature before becoming live (256 blocks, ~17 hours)
+Tickets missed               | Tickets that missed a vote either because the wallet was offline or the PoW miner didn't mine it properly.
+Tickets revoked              | Missed tickets are revoked and the ticket price is refunded to you (minus the ticket fee).
```

Comment:
I have no idea how to interpret this description because it is the only one in this entire table that is a complete sentence rather than a clarification clause.